apache-rat-tasks/pom.xml

```diff
@@ -35,6 +35,11 @@
       <artifactId>apache-rat-core</artifactId>
       <version>${project.parent.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.rat</groupId>
+      <artifactId>apache-rat-tools</artifactId>
+      <version>${project.parent.version}</version>
```

Comment:
Should this be the same as project.version as we do not release parts of RAT but all as. a big bunch.